crates/escrow/src/lib.rs

```diff
@@ -471,6 +475,8 @@ impl<T: Config> Pallet<T> {
             new_locked.end = unlock_height;
         }
 
+        T::EscrowRewards::deposit_stake(who, amount)?;
```

Comment:
This gives people a stake equal to their KINT rather than their vKint, right? That is not what the docs describe:

```
The staking rewards are distributed on a per-block basis, proportoinal to the vKINT balance per user (using interBTC’s scalable reward distribution mechanism). This means: to keep receiving the same amount of rewards, vKINT locks must be extended periodically.
```

Reply:
Users would only receive rewards once they lockup KINT which is a prerequisite to minting vKINT. I see no easy way to implement rewards based on linearly degrading escrow balances.

Reply:
> Users would only receive rewards once they lockup KINT which is a prerequisite to minting vKINT. I see no easy way to implement rewards based on linearly degrading escrow balances.

Yea I agree, I also don't see it. But if you're diverting from docs/spec, then @nud3l or @alexeiZamyatin should approve

Reply:
Note also that we remove all stake on withdraw, so it incentivizes continued lockup.

Reply:
Let's keep the code as-is for now. Then we'll fix this early in January before we activate the stake to vote rewards. Since the rewards are anyway not active, this should not cause an issue now.